integration/test/cloudwatchlogs/resources/write_logs.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh
+
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: MIT
+
+n="$1"
+if [ -z "$1" ] || [ $n -le 1 ]; then
+  n=100
+fi
+file_path="$2"
+if [ -z "$2" ]; then
+  file_path='/tmp/test.log'
+fi
+
+echo "Emitting $n logs to $file_path"
+
+for i in $(seq 1 $n); do
+  echo "$(date -Is) [foo] This is a log line" >>$file_path
+  echo "$(date -Is) [bar] This is a log line" >>$file_path
+  sleep 1 # sleep so the timestamps for the logs are different
+done
```

Comment:
You could delete this shell script and write it in Golang.
In general I would recommend sticking with the primary programming language of the code repo if/when possible.
Then you could use Golang asserts on opening and writing the file.
Another reason would be potentially reusing the same integration test on Windows EC2 instances.

Reply:
That's a good point. This along with @ZhenyuTan-amz 's comment, I think I'll rewrite it. Good call out on propping this up for Windows integration tests. I was thinking of duplicating the script but this is better 😛